R-Project/R/fpRerF.R

```diff
@@ -31,7 +38,7 @@
 
 
 fpRerF <-
-	function(X=NULL, Y=NULL,csvFileName=NULL, columnWithY=NULL, maxDepth = Inf, minParent=1, numTreesInForest=500, numCores=1,numTreeBins=NULL, forestType="binnedBaseRerF", nodeSizeToBin=NULL, nodeSizeBin=NULL,mtry=NULL, mtryMult=NULL,seed=sample(1:1000000,1)){
+	function(X=NULL, Y=NULL,csvFileName=NULL, columnWithY=NULL, maxDepth = Inf, minParent=1, numTreesInForest=500, numCores=1,numTreeBins=NULL, forestType="binnedBaseRerF", nodeSizeToBin=NULL, nodeSizeBin=NULL,mtry=NULL, mtryMult=NULL,seed=sample(1:1000000,1), methodToUse = 1, imageHeight = NULL, imageWidth = NULL, patchHeightMax = NULL, patchHeightMin = NULL, patchWidthMax = NULL, patchWidthMin = NULL){
```

Comment:
i think structured rerf should be able to be set in the "forest type" no?  then if that's set to "s-rerf" we set `methodToUse`...

Reply:
Good point, that'd be much more clear.